src/main/java/bean/task/TodoTask.java

```diff
@@ -0,0 +1,47 @@
+package bean.task;
+
+/**
+ * Represents a todo task. Inherits from the Task class.
+ */
+public class TodoTask extends Task {
+
+    /**
+     * Initializes a TodoTask with the specified name.
+     *
+     * @param name The name of the todo task.
+     */
+    public TodoTask(String name) {
```

Comment:
I like how each class is named with Task at the end to show that they inherit from Task!